opensrp-anc/src/main/assets/json.form/anc_register.json

```diff
@@ -292,6 +293,10 @@
         "v_required": {
           "value": "true",
           "err": "{{anc_register.step1.phone_number.v_required.err}}"
+        },
+        "v_max_length": {
+          "value": "12",
+          "err": "Phone number characters cannot exceed 12 characters"
```

Comment:
Is this a requirement for the implementation?

Reply:
Yes. This is an issue that was requested, was tested and passed QA and thought of merging it here.